pyproject.toml

```diff
@@ -9,32 +9,30 @@ repository = "https://github.com/moj-analytical-services/splink"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<4.0.0"
+python = ">=3.8.0,<4.0.0"
 jsonschema = ">=3.2,<5.0"
 # 1.3.5 is the last version supporting py 3.7.1
 pandas = ">1.3.0"
 duckdb = ">=0.8.0"
 sqlglot = ">=13.0.0, <19.0.0"
 altair = "^5.0.1"
 Jinja2 = ">=3.0.3"
-phonetics = "^1.0.5"
+phonetics = ">=1.0.5"
```

Comment:
I'll try and produce our own metaphone algo to remove this dependency at some point this week.